### ISSUE TITLE
WA doesn't load under "Battleground" size trigger until I type /WA

### ISSUE THREAD

**Issue by Celeriestix** [WeakAuras/WeakAuras2#5182]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues.

### Description

https://streamable.com/gh68tm

Video illustration of bug above. Retested on Alpha version 5.14.0-8-g01420f6 and still not working.

Error: I've created a WA to track an item cooldown and load as "Instance Size Type > Battlegrounds". Upon joining a BG, the WA does not work until I type /WA and close the menu. Using the item does not trigger the WA to activate either.

### WeakAuras Version

5.14

### World of Warcraft Flavor

Retail (Default)

### World of Warcraft Region

US/NA

### Tested with only WeakAuras

I got this issue with only WeakAuras enabled

### Reproduction Steps

1. New Aura.
2. Icon.
3. Trigger > Item > Cooldown progress Item
4. Item > Frostweave net (or anything)
5. Show > Always
6. Load
7. Instance Size Type > battlegrounds.

**Comment by InfusOnWoW** [comment]
I cannot reproduce the issue. Please install bugsack + buggraber to check for any lua errors.

Also the issue template asked you to confirm the bug without any other addons installed, did you try that?

**Comment by Celeriestix** [comment]
> I cannot reproduce the issue. Please install bugsack + buggraber to check for any lua errors.
> 
> Also the issue template asked you to confirm the bug without any other addons installed, did you try that?

Hey, thanks for the fast response, Infus. 

You're right, I missed that. I removed all addons and unloaded any other Weakauras and was able to replicate this with a caveat:

The issue only happens when more than one Weakaura is set to load instance size battleground. 

If I only have ONE Weakaura it works. More than one? It only loads one or two.

The steps to reproduce this are identical to the ones in my original post. Simply create several Weakauras set to load instance size battleground. Upon entering a BG, only one or two will load.

I've showcased this error in another video: https://streamable.com/0hlljx
